Simplified/Accounts.swift

```diff
@@ -13,7 +13,7 @@ final class AccountsManager: NSObject
   static let shared = AccountsManager()
   
   // For Objective-C classes
-  @objc class func sharedInstance() -> AccountsManager
+  @objc @discardableResult class func sharedInstance() -> AccountsManager
```

Comment:
This is my fault based on how the method is being used in the app delegate. I'm not saving the singleton the first time we run it.
@winniequinn Let me know a better way to do this.. I see an example of running calling `initialize` but I'm not sure the behavior here would be equivalent.

Reply:
@gioneill: It's not obvious to me that calling `AccountsManager.sharedInstance` has an effect. What is the reason we're presently invoking it here?

https://github.com/NYPL-Simplified/Simplified-iOS/blob/adaef5ec3ad0bf56889cfbf5dc2875bf9ecd2c2b/Simplified/NYPLAppDelegate.m#L42

Reply:
In any case, we do not want to mark `sharedInstance` as discardable. Doing so may mask a bug in our code because the only reason to get the shared instance should be to do something with it.

Reply:
You're right @winniequinn! Line 42 and any comment above it should be deleted.

Reply:
@vuinguyen: We do not want to mark it as discardable, no. Instea,d we want to change the interface of the class so it no longer requires that we discard the result of `sharedInstance` to initialize the class in the first place.

I spoke with @gioneill about this and I think we're both on the same page. Please work with him to get this fixed!